cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -203,32 +202,29 @@ def is_disposable_income_complete(case_data):
 
     @staticmethod
     def is_property_complete(case_data):
-        if not hasattr(case_data.facts, "has_partner"):
+        if not "has_partner" in case_data.facts.__dict__:
             # If they have a partner then that may increase assets that they need to delare, so
             # this section is not complete until we clear up if there is a partner
             return False
 
         if len(case_data.property_data) == 0:
             return True
-        for key, value in case_data.property_data[0].iteritems():
+        for key, value in case_data.property_data[0].items():
```

Comment:
Interesting. I thought iteritems() was preferred in python2, because it does it generatively, even though in python3 items() has the generative behaviour too, so is then preferred

Reply:
My IDE showed this to me and said it was incompatible with python 3. It seems fine, we only have a handful of items so generator vs list seems pretty unimportant

Reply:
agreed it's unimportant, but the original is still better, so I'd argue not helpful to change it in this PR